packages/maskbook/src/plugins/RedPacket/UI/RedPacketDialog.tsx

```diff
@@ -33,7 +33,7 @@ export default function RedPacketDialog(props: RedPacketDialogProps) {
                     payload.password = prompt('Please enter the password of the red packet:', '') ?? ''
                 }
 
-                if (payload.contract_version === 2) {
+                if (payload.contract_version !== 1) {
```

Comment:
plz add a comment.

Reply:
why not 

```suggestion
                if (payload.contract_version > 1) {
```

if `payload.contract_version` equals `0`?